metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchUtilImpl.java

```diff
@@ -90,13 +90,14 @@
     private final MetacatJson metacatJson;
     private XContentType contentType = Requests.INDEX_CONTENT_TYPE;
     private final ElasticSearchMetric elasticSearchMetric;
+
     /**
      * Constructor.
      *
-     * @param client      elastic search client
-     * @param config      config
-     * @param metacatJson json utility
-     * @param elasticSearchMetric    elastic search metric
+     * @param client              elastic search client
+     * @param config              config
+     * @param metacatJson         json utility
+     * @param elasticSearchMetric elastic search metric
      */
     public ElasticSearchUtilImpl(
         @Nullable final Client client,
```

Comment:
Does the new ES client have retry logic? If so, can we use that instead of us handling the retry logic.

Reply:
No it doesn't have the retry logic.